lib/services/geocoder.js

```diff
@@ -119,15 +128,26 @@ MapboxGeocoder.prototype.geocodeReverse = function(location, options, callback)
     dataset = options.dataset;
   }
 
+  var precision = REVERSE_GEOCODER_PRECISION;
+  if (options.precision) {
+    assert(typeof options.dataset === 'number', 'precision option must be number');
+    precision = options.precision;
+  }
+
   this.client({
     path: constants.API_GEOCODER_REVERSE,
     params: {
-      longitude: location.longitude,
-      latitude: location.latitude,
+      longitude: roundTo(location.longitude, precision),
+      latitude: roundTo(location.latitude, precision),
       dataset: dataset
     },
     callback: callback
   });
 };
 
+function roundTo(value, places) {
```

Comment:
it's pretty close, but I don't think it's desirable to pad with extra zeroes (e.g. `2.00000`) for cache hit rate purposes.